spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/Binding.java

```diff
@@ -37,4 +39,17 @@
 	 * and a new Binding should be created instead.
 	 */
 	void unbind();
+
+	/**
+	 * Binds the target component represented by this instance. Should be used to call any
+	 * lifecycle method on the represented target. Implementations must be idempotent.
+	 * This method is called after the concrete {@link Binder} implementation returns from
+	 * its create binding methods
+	 */
+	void bind();
+
+	/**
+	 * @return the binding target instance
+	 */
+	T getTarget();
```

Comment:
It's meant to be used by the implementations on the `configure` step of the `BinderErrorConfigurer`. MessageChannel binders are a bit odd in that sense since we need to configure the endpoint and not the target. But the reactive binder for instance you would need to configure the target